### DISCUSSION TITLE
dynamic update of configuration

### DISCUSSION THREAD

**Discussion by hknath** [open-telemetry/opentelemetry-collector#7450] [Q&A]
Hi 
Is it possible to update the the configuration either receiver, processor or exporter, without restarting the collector pod

**Comment by garry-cairns** [comment]
I've not seen it done but [this line](https://github.com/open-telemetry/opentelemetry-collector/blob/05d57d966f2afe7cc139cdac87e2c768c2a125b5/otelcol/collector.go#L69) suggests to me it's possible.
